project_forecast_line_bokeh_chart/report/forecast_line_reporting.py

```diff
@@ -52,8 +49,8 @@ def _compute_bokeh_chart(self):
         # period.
         plots = self._build_plots()
         grid = column(*plots)
-        script, div = components(grid, wrap_script=False)
-        self.bokeh_chart = json.dumps({"div": div, "script": script})
+        script, div = components(grid)
+        self.bokeh_chart = "%s%s" % (div, script)
```

Comment:
I've [forwarded ported the new bokeh](https://github.com/sunflowerit/web/pull/8/files), so it would also be an option to depend on that PR instead of doing this.